Gopkg.toml

```diff
@@ -14,3 +14,7 @@
 [[constraint]]
   branch = "v2"
   name = "gopkg.in/hairyhenderson/yaml.v2"
+
+[[constraint]]
+  name = "github.com/zealic/xignore"
+  version = "0.3.2"
```

Comment:
is there a reason you're pinning to this release? this shouldn't be necessary in this case - there's a tag and `dep` will understand it just fine...

Reply:
I think minor range version `~` is better than fixed version.
I will maintain upstream repo to keep stability

I will maintain upstream repo to keep stablelity.

## See also:
* https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md#version
* https://semver.org